webapp/src/main/resources/sass/mojito.scss

```diff
@@ -167,6 +167,7 @@ a.label-container:hover {
 /** MODAL */
 .modal-content {
     box-shadow: 0 2px 6px rgba(0, 0, 0, 0.1);
+    word-wrap: break-word;
```

Comment:
isn't that the default value? Do we need it/can we remove it?

Reply:
looks like it can be removed. There was text spilling out of the modal before, but it was probably caused by something else.